app/styles/main.scss

```diff
@@ -24,6 +24,12 @@ $icon-font-path: "/bower_components/bootstrap-sass-official/assets/fonts/";
     width: 250px;
 }
 
+.zipcodeValidCheck{
```

Comment:
naming convention: Always use dashed names for css classes.

Here use a generic class name. I think since we are using bootstrap CSS, we can use the `alert-success` class.